### DISCUSSION TITLE
get orientation of image from meta data

### DISCUSSION THREAD

**Discussion by YingdiZhang** [Project-MONAI/MONAI#4224] [Q&A] [answered]
Thanks for the great job!
I wonder how to get  the orientation of a *.nii file , like RAS or LPI.
I have get the meta information refering to https://github.com/Project-MONAI/MONAI/discussions/3727, but dont found the orientation information in the meta data

**Comment by rijobro** [comment, marked as answer]
If you have the affine transformation matrix (e.g., `data["image_meta_dict"]["affine"]`), you can get the orientation with this:

```python
affine = np.eye(4)  # or e.g., data["image_meta_dict"]["affine"]
print("".join(nib.aff2axcodes(affine)))
```

This is the relevant bit of code in our transform: https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/spatial/array.py#L531.
